infra/feast-operator/internal/controller/featurestore_controller_test_utils_test.go

```diff
@@ -104,14 +104,38 @@ func createFeatureStoreResource(resourceName string, image string, pullPolicy co
 		Spec: feastdevv1alpha1.FeatureStoreSpec{
 			FeastProject: feastProject,
 			Services: &feastdevv1alpha1.FeatureStoreServices{
+				Volumes: []corev1.Volume{
+					{
+						Name: "config-volume",
+						VolumeSource: corev1.VolumeSource{
+							ConfigMap: &corev1.ConfigMapVolumeSource{
+								LocalObjectReference: corev1.LocalObjectReference{
+									Name: "my-config",
+								},
+							},
+						},
+					},
+				},
```

Comment:
rather than add this to the main `createFeatureStoreResource`, maybe this would do better as an after the fact addition within a separate unit test file?